src/main/kotlin/calculator/StringCalculator.kt

```diff
@@ -0,0 +1,48 @@
+package calculator
+
+class StringCalculator {
+
+    fun calculate(expression: String?): Double {
+        validateExpression(expression)
+
+        val tokens = expression!!.split(" ")
```

Comment:
I think this code is dangerous because calculate receives a nullable expression and uses the !! operator inside the method. What do you think?

Reply:
What problems can these situations cause?